README.md

```diff
@@ -1,596 +1,493 @@
 OpenST.js
 ============
 
-OpenST is a framework for building token economies. Here are steps to get started with OpenST.js. In order to view an example, please visit our example repo, [OpenST.js Examples](https://github.com/OpenSTFoundation/openst-js-examples).
+![Build master](https://img.shields.io/travis/OpenSTFoundation/openst.js/master.svg?label=build%20master&style=flat)
+![Build develop](https://img.shields.io/travis/OpenSTFoundation/openst.js/develop.svg?label=build%20develop&style=flat)
+![npm version](https://img.shields.io/npm/v/@openstfoundation/openst.js.svg?style=flat)
+![Discuss on Discourse](https://img.shields.io/discourse/https/discuss.openst.org/topics.svg?style=flat)
+![Chat on Gitter](https://img.shields.io/gitter/room/OpenSTFoundation/SimpleToken.svg?style=flat)
 
-#### Overview of different components:
```

Comment:
this high level description (with minor update to the TokenHolder) is quite valuable to have at the top of the readme

Reply:
I think this should be part of openst-contracts as that seems to be ideal placeholder 🤔 . 
We can point to openst-contracts readme through github url.